benchmarks/shelley3pools/benchmark.sh

```diff
@@ -27,8 +27,22 @@ set -e
 rm -rf ./db/* ./logs/*
 mkdir -p db logs/sockets
 
-# 1 prepare genesis
-./prepare_genesis_staked.sh
+# 1 prepare geneses for all eras & update configs
+rm -f 'configuration/start-time'
+. "$BASEDIR"/configuration/parameters
+./prepare_genesis_byron.sh
```

Comment:
for pure Shelley we do not need a byron genesis, right?

Reply:
We don't, but it doesn't hurt either -- adding a branch here would only complicate the code.